site-src/geps/gep-1364.md

```diff
@@ -0,0 +1,328 @@
+# GEP-1364: Status and Conditions Update
+
+* Issue: [#1364](https://github.com/kubernetes-sigs/gateway-api/issues/1364)
+* Status: Implementable
+
+(See definitions in [Kubernetes KEP][kep-status].
+
+
+
+## TLDR
+
+The status, particularly the Conditions, across the whole Gateway API has very much grown
+grown organically, and so has many inconsistencies and odd behaviors.
+This GEP covers doing a review and consolidation to make Condition behavior consistent
+across the whole API.
+
+## Goals
+
+* Update Conditions design to be consistent across Gateway API resources
+* Provide a model and guidelines for Conditions for future new resources
+* Specify changes to conformance required for Condition updates
+
+## Non-Goals
+
+
+
+## Introduction
+
+Gateway API currently has a lot of issues related to status, especially that
+status is inconsistent ([#1111][1111]), that names are hard to understand ([#1110][1110]),
+Reasons aren't explained properly ([#1362][1362]), and so on.
+
+As the API has grown, the way we talk about resources has changed a lot, and some of the
+status design hasn't been updated since resources were created.
+
+So, for example,  we have GatewayClass with `Accepted`, Gateway with `Scheduled`,
+the Gateway Listeners with `Detached` (which you want to be `false`, unlike the previous
+two), and then Gateways and Gateway Listeners have `Ready`, but Route doesn't (and which
+also you want to be `true`).
+
+This document lays out large-scale changes to the way that we talk about resources,
+and the Conditions to match them. This means that there will be an unavoidable break
+in what constitutes a healthy or unhealthy resource, and there will be changes
+required for all conformant implementations to stay conformant.
+
+This level of change is not optimal, and the intent is to make this a one-off change
+that can be built upon for future resources - since there are definitely more resources
+on the way.
+
+## Background: Kubernetes API conventions and prior art on Conditions
+
+Because this GEP is mainly concerned with updating the Conditions we are setting in
+Gateway API resources' `status`, it's worth reviewing some important points about
+Conditions. (Thi information is mainly taken from the [Typical status properties][typstatus]
```

Comment:
Missing href for the link here?

Reply:
It's all the way down at the bottom of the doc, with the other references. I thought it was better to have everything in the one place.